application/core/plugins/ExportSTATAxml/STATAxmlWriter.php

```diff
@@ -401,7 +401,7 @@ protected function updateCustomresponsemap()
                             // For radio lists, user wants code, not label
                             // TODO: We could skip this loop if we had answer code
                             foreach ($this->customFieldmap['answers'][$iQID][$iScaleID] as $answer) {
-                                if ($answer['answer'] == $response) {
+                                if (is_array($answer) && isset($answer['answer']) && $answer['answer'] == $response) {
```

Comment:
`if (isset($answer['answer']) && $answer['answer'] == $response) {`

Check if it's an array not needed, just check if it exist.

Reply:
Thank you! As requested I removed the array check.